composer/trainer/trainer.py

```diff
@@ -1187,6 +1187,27 @@ def saved_checkpoints(self) -> List[str]:
             return []
         return self._checkpoint_saver.saved_checkpoints
 
+    def _attempt_checkpoint_download(self, save_folder: str, latest_checkpoint_path: str,
```

Comment:
nit: `_try_checkpoint_download` ? 

also, perhaps move the `os.makedirs` outside so this function does one thing.